### PR TITLE
send http headers in the Get request 

### PR DESCRIPTION
**Pull request by acer618**
Need help with these 2 errors:

Building fullerite...
# fullerite/collector
src/fullerite/collector/nerve_uwsgi.go:74:23: cannot use serviceHeadersMap (type interface {}) as type map[string]map[string]string in assignment: need type assertion
src/fullerite/collector/nerve_uwsgi.go:170:24: too many arguments in call to client.Get
	have (string, map[string]string)
	want (string)

### CODE REVIEW

File: src/fullerite/collector/nerve_uwsgi.go [hunk 1]
```diff
@@ -51,6 +52,7 @@ func newNerveUWSGI(channel chan metric.Metric, initialInterval int, log *l.Entry
 	col.name = "NerveUWSGI"
 	col.configFilePath = "/etc/nerve/nerve.conf.json"
 	col.queryPath = "status/metrics"
+	//col.serviceHeadersMap = make(map[string]map[string]string)
```
Comment (drolando) [inline src/fullerite/collector/nerve_uwsgi.go]: delete

File: src/fullerite/collector/nerve_uwsgi.go [hunk 2]
```diff
@@ -218,7 +240,9 @@ func (n *nerveUWSGICollector) tryFetchUWSGIWorkersStats(serviceName string, endp
 	emptyResult := []metric.Metric{}
 	serviceLog := n.log.WithField("service", serviceName)
 	serviceLog.Debug("making GET request to ", endpoint)
-	rawResponse, _, err := queryEndpoint(endpoint, n.timeout)
+	headers := n.serviceHeadersMap[serviceName]
```
Comment (drolando) [inline src/fullerite/collector/nerve_uwsgi.go]: what happens if there are no headers for that service name? Will this crash?
Can you add a test for this?

File: src/fullerite/collector/http_dropwizard.go [hunk 3]
```diff
@@ -81,7 +81,8 @@ func (h *httpDropwizardCollector) queryService(s ServiceEndpoint) {
 	endpoint := fmt.Sprintf("http://localhost:%s/%s", s.Port, s.Path)
 	serviceLog.Debug("making GET request to ", endpoint)
 
-	rawResponse, schemaVer, err := queryEndpoint(endpoint, h.timeout)
+	headers := make(map[string]string)
```
Comment (drolando) [inline src/fullerite/collector/http_dropwizard.go]: nit: I think you can also pass directly `map[string]string{}` which will save you one variable creation. Not a big deal though

File: src/fullerite/collector/nerve_uwsgi.go (hunk 2 above)
Comment (acer618) [reply src/fullerite/collector/nerve_uwsgi.go]: added a check. will create a backlog ticket for the test

**Pull request merged**